components/shell/src/shell/shell.tsx

```diff
@@ -98,7 +98,9 @@ export const Shell: FC<ShellProps> = ({
 
         <div
           className={
-            !isTabletLayout ? mergeClasses(classes.grid, classes.minHeight) : undefined
+            !isTabletLayout
```

Comment:
Nit (non-blocking): I find it a bit easier to look at conditionals like this as affirmations. So something like `isTabletLayout ? classes.minHeightTablet : mergeClasses(classes.grid, classes.minHeight)` is easier to read IMHO.

Reply:
💯